pkg/cloud/services/eks/cluster.go

```diff
@@ -207,6 +211,18 @@ func (s *Service) setStatus(cluster *ekstypes.Cluster) error {
 			conditions.MarkFalse(s.scope.ControlPlane, ekscontrolplanev1.EKSControlPlaneUpdatingCondition, "updated", clusterv1.ConditionSeverityInfo, "")
 			record.Eventf(s.scope.ControlPlane, "SuccessfulUpdateEKSControlPlane", "Updated EKS control plane %s", s.scope.KubernetesClusterName())
 		}
+		if s.scope.ControlPlane.Spec.UpgradePolicy == ekscontrolplanev1.UpgradePolicyStandard &&
+			(specSemver.Major < clusterSemver.Major || specSemver.Minor < clusterSemver.Minor) {
```

Comment:
this will make the code slightly easier

```suggestion
			specSemver.LT(clusterSemver) {
```

Reply:
Reference here https://pkg.go.dev/github.com/blang/semver#Version.LT

Reply:
Or maybe there's a reason you're ignoring the patch version here. If so please let me know! 

also wouldn't this return true in a case where specSemver == 2.1.0 and clusterSemver == 1.2.0